source/cxx.txt

```diff
@@ -17,8 +17,11 @@ MongoDB C++ Driver
 Introduction
 ------------
 
-The MongoDB C++ Driver is the official client library for C++ applications
-using the C++11 (or later) standard.
+Welcome to the documentation site for the official MongoDB C driver. 
+You can add the driver to your application to work with MongoDB using 
+the C++11 (or later) standard. Download the library, ``mongocxx``,  
```

Comment:
We should avoid parentheses except for stating abbreviations/abbreviation expansions.
See: https://docs.mongodb.com/meta/style-guide/style/punctuation/#parentheses
```suggestion
the C++11 or later standard. Download the library, ``mongocxx``,  
```